sql/core/src/test/scala/org/apache/spark/sql/CollationStringExpressionsSuite.scala

```diff
@@ -73,6 +74,57 @@ class CollationStringExpressionsSuite extends QueryTest
     })
   }
 
+  test("TRANSLATE check result on explicitly collated string") {
+    def testTranslate(input: String,
```

Comment:
this should probably be just a private method, no need to wrap it inside a test since the the test wouldn't be testing anything